source/common/quic/quic_transport_socket_factory.h

```diff
@@ -46,7 +46,7 @@ class QuicTransportSocketFactoryBase : public Network::CommonTransportSocketFact
   // To be called right after construction.
   virtual void initialize() PURE;
 
-  // Network::TransportSocketFactory
+  // Network::UpstreamTransportSocketFactory
```

Comment:
move this to QuicClientTransportSocketFactory?

Reply:
Done.